leaf-ui/plugins/slider/nouislider.min.css

```diff
@@ -1,4 +1,316 @@
 /*! nouislider - 8.2.1 - 2015-12-02 21:43:15 */
 
+.noUi-target,.noUi-target *
+{   -webkit-touch-callout:none;
+    -webkit-user-select:none;
+    -ms-touch-action:none;touch-action:none;
+    -ms-user-select:none;
+    -moz-user-select:none;
+    -moz-box-sizing:border-box;
+    box-sizing:border-box

```

Comment:
Do we need a `- ` in the front here as well?

Reply:
I am not sure. I just made the file more readable. I didn't write it.